install/kubernetes/helm/istio/values-istio-all.yaml

```diff
@@ -8,6 +8,9 @@ global:
 sidecar-injector:
   enabled: true
 
+galley:
+  enabled: true
```

Comment:
Should it be enabled by default in 0.8 ? What are the dependencies / setup / etc ?

Reply:
k8s >= 1.9

Config validation is targeted BETA for 1.0. I could disable for 0.8 but would like to (re)enable for 0.9 for additional testing prior to 1.0.